docs/packaging.md

```diff
@@ -21,14 +21,12 @@ WORKDIR /workspace
 # copy the source to the docker image
 COPY . /workspace
 
-RUN swift build -c release
+RUN swift build -c release --static-swift-stdlib
 
 #------- package -------
-FROM centos:8
+FROM centos8
```

Comment:
Typo here?
No `centos8` repo available yet, we have:
```suggestion
FROM centos:centos8
```

Reply:
Thanks!